.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,24 @@
+# .github/workflows/chromatic.yml
+
+name: 'Chromatic'
+
+on: push
```

Comment:
I think it should be done at the PR level. So, you should run this workflow as per `pull_request_target`.

Reply:
Okay Sure